### DISCUSSION TITLE
Custome alert name

### DISCUSSION THREAD

**Discussion by oliverhavrila** [Security-Onion-Solutions/securityonion#10509] [General]
Dear SO,

great product, I really like it. I changed 'standalone' architecture to 'distributed' architecture and I am thinking about..

Is possible in every forwarded node set up custom name before rule.name? For example if I have 5 offices I would like to see, from which offices that alert comes: [for example](https://anonymfile.com/OEX0J/custome-name.png)

Thank you

**Comment by dougburks** [comment]
Have you tried clicking the drop-down box and selecting the second query to group by sensor?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/670a157d-3d73-4a41-9ce8-b732e4ebe8f0)

For further information, please see:
https://docs.securityonion.net/en/2.3/alerts.html#query-bar
